Userland/Libraries/LibGfx/AffineTransform.h

```diff
@@ -25,6 +25,8 @@ class AffineTransform {
     {
     }
 
+    [[nodiscard]] static AffineTransform translation(FloatPoint);
```

Comment:
Do we have a dependency between FloatPoint and AffineTransform, or why is this out of line

Reply:
This commit is no longer here, I found Painter.translate() in the meantime. (But it was out of line because the .h didn't include Point.h)